File: Daily Challanges/2026-05-May/2026-05-31.js
function getCombinations(n) {
  let result = 1;
  for (let i = 0; i < n; i++) {
    result = (result * (2 * n - i)) / (i + 1);
  }
  return Math.round(result / (n + 1));
}

console.log(getCombinations(0)); // 1
console.log(getCombinations(1)); // 1
console.log(getCombinations(2)); // 2
console.log(getCombinations(3)); // 5
console.log(getCombinations(4)); // 14

File: Daily Challanges/2026-05-May/2026-05-31.py
from math import comb
def get_combinations(n):
    return comb(2 * n, n) // (n + 1)

print(get_combinations(0))  # 1
print(get_combinations(1))  # 1
print(get_combinations(2))  # 2
print(get_combinations(3))  # 5
print(get_combinations(4))  # 14